cylc/uiserver/resolvers.py

```diff
@@ -317,9 +317,13 @@ async def play(cls, workflows, args, workflows_mgr, log):
                 if ret:
                     # command failed
                     _, err = proc.communicate()
+                    msg = err.strip() or (
```

Comment:
I believe it would more helpful than not to include stdout if there is no stderr

```suggestion
                    out, err = proc.communicate()
                    msg = err.strip() or out.strip() or (
```